src/app/components/SidePanel/SidePanel.tsx

```diff
@@ -32,6 +32,8 @@ interface SidePanelProps {
 	totalSteps?: number;
 	activeStep?: number;
 	footer?: React.ReactNode;
+	onBack?: () => void;
+	lastStep?: boolean;
```

Comment:
```suggestion
	isLastStep?: boolean;
```
since it's a boolean, better to use `is` or `has` in front to indicate that